bundles/org.openhab.binding.keba/src/main/resources/OH-INF/update/kecontact-enabled.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="keba:kecontact">
+		<instruction-set targetVersion="1">
+			<remove-channel id="enabled"/>
+			<add-channel id="enabledsystem">
+				<type>keba:enaledsystem</type>
+				<label>Enabled (System)</label>
```

Comment:
No need to set the label.

Reply:
Labels is set for all channels in this binding. I assume it's good to have no exception here for uniformity.

Reply:
Label of course is set in thing description. Here we are in upgrade instructions and redefining a label is useless.

Reply:
It will also overwrite any localized label.

Reply:
Label removed.